nomad/grapl-core.nomad

```diff
@@ -0,0 +1,428 @@
+        # Spins up a "canary" instance of potentially destructive updates, validates that they are healthy, then promotes the instance to update
+        canary = 1
+        max_parallel = 1
+        # The min amount of reported "healthy" time before a instance is considered healthy and an allocation is opened up for further updates
+        min_healthy_time = "15s"
+    }
+
+    group "dgraph-zero-leader" {
+        network {
+            mode = "bridge"
+        }
+
+        task "dgraph-zero-leader" {
+            driver = "docker"
+
+            config {
+                image = "dgraph/dgraph:latest"
+                args = [
+                    "dgraph",
+                    "zero",
+                    "--my=localhost:5080",
+                    "--replicas", "${var.dgraph_replicas}",
+                    "--raft", "idx=1",
+                ]
+            }
+        }
+
+        service {
+            name = "dgraph-zero-leader-grpc-private"
+            port = "5080"
+
+            connect {
+                sidecar_service {
+                    proxy {
+                        # Connect the Zero leader to the Zero followers
+                        dynamic "upstreams" {
+                            iterator = zero_follower
+                            for_each = local.dgraph_zeros
+
+                            content {
+                                destination_name = "dgraph-zero-follower-${zero_follower.value.id}-grpc-private"
+                                local_bind_port = zero_follower.value.grpc_private_port
+                            }
+                        }
+
+                        # Connect this Zero leader to the Alphas
+                        dynamic "upstreams" {
+                            iterator = alpha
+                            for_each = [for alpha in local.dgraph_alphas: alpha]
+
+                            content {
+                                destination_name = "dgraph-alpha-${alpha.value.id}-grpc-private"
+                                local_bind_port = alpha.value.grpc_private_port
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    # Create DGraph Zero followers
+    dynamic "group" {
+        iterator = zero
+        for_each = local.dgraph_zeros
+        labels = ["dgraph-zero-follower-${zero.value.id}"]
+
+        content {
+            network {
+                mode = "bridge"
+            }
+
+            task "dgraph-zero-follower" {
+                driver = "docker"
+
+                config {
+                    image = "dgraph/dgraph:latest"
+                    args = [
+                        "dgraph",
+                        "zero",
+                        "--my=localhost:${zero.value.grpc_private_port}",
+                        "--replicas", "${var.dgraph_replicas}",
+                        "--raft", "idx=${zero.value.id + 1}",
+                        "-o", "${zero.value.id}",
```

Comment:
Is there a long option form for `-o` that we can use?

Reply:
I don't know offhand, but I suspect that it'll probably be `--offset`. I'll look and see if I can update it :)

Reply:
It turned out to be `--port_offset`; Updated :)